planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -925,7 +813,7 @@ bool PullOverModule::isStopped()
 
 bool PullOverModule::hasFinishedCurrentPath()
 {
-  const auto current_path_end = status_.path.points.back();
+  const auto current_path_end = getCurrentPath().points.back();
```

Comment:
Better to use reference since this is just alias

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/1911/commits/b47d458c9870cfcb74d01d9866159bd273ee1972